tests/distinct_generators/test_categorical_generator.py

```diff
@@ -9,7 +9,7 @@
 class TestRandomsCategories(unittest.TestCase):
     def setUp(self):
         self.rng = Generator(PCG64(12345))
-        self.categories = ["People", "Cats", "Dogs"]
+        self.categories = ["People", "Cats", "Dogs", "Bees", "Spiders"]
```

Comment:
```suggestion
        self.categories = ["Lions", "Tigers", "Bears", "Oh my"]
```
missed opportunity.... sadness

Reply:
goofin'